better_uptime.py

```
@@ -0,0 +1,89 @@
+    
+    try:
+        response = requests.post(url, headers=headers, json=payload, timeout=10)
+        if response.ok:
+            print(f"🔔 [BU ALERT SENT] {summary}")
+        else:
+            print(f"🔴 Failed to send BU alert: {response.status_code} {response.text}")
+    except Exception as e:
+        print(f"🔴 Exception sending BU alert: {e}")
+
+def fetch_open_bu_incidents():
+    url = getattr(cfg, "BETTER_UPTIME_URL")
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)